examples/manual-approve-with-azure-devops/azure-pipelines.yml

```diff
@@ -81,3 +81,20 @@ steps:
   env:
     ARM_CLIENT_SECRET: $(ARM_CLIENT_SECRET)
     DATABRICKS_TOKEN: $(DATABRICKS_TOKEN)
+
+- task: PowerShell@2
+  displayName: Update Pull Request
+  inputs:
+    targetType: 'inline'
+    script: |
+      $pullRequestId = $env:SYSTEM_PULLREQUEST_PULLREQUESTID
+      $repositoryId = $env:BUILD_REPOSITORY_ID
+      $projectName = $env:SYSTEM_TEAMPROJECT
+      $organizationUri = $env:SYSTEM_COLLECTIONURI
+      $buildId = $env:BUILD_BUILDID
+      $terraformPlanCommand = "terraform plan -lock=false -input=false -no-color"    
```

Comment:
can we capture `terraform plan` output earlier, and store it in the file on disk, and then reuse it?  To avoid doing it twice...